packages/discord.js/src/structures/Embed.js

```diff
@@ -2,11 +2,19 @@
 
 const { Embed: BuildersEmbed } = require('@discordjs/builders');
 const Transformers = require('../util/Transformers');
+const { Util } = require('../util/Util');
 
 class Embed extends BuildersEmbed {
   constructor(data) {
     super(Transformers.toSnakeCase(data));
   }
+
+  setColor(color) {
```

Comment:
this should be documented because the allowed type is different (also the main class should be documented too ig)